src/lib/manage-ad-free-cookie.ts

```diff
@@ -11,7 +11,6 @@ const AD_FREE_USER_COOKIE = 'GU_AF1';
  * 3. The url parameter noadsaf is set so ads are temporarily disabled
  */
 enum AdFreeCookieReasons {
-	ConsentOptOut = 'consent_opt_out',
 	Subscriber = 'subscriber',
 	ForceAdFree = 'force_ad_free',
```

Comment:
If `ForceAdFree` can be [determined by `window.location.hash`](https://github.com/guardian/commercial/blob/4ccf6f96b86c05c11faa2baeb53ade7856ad1255/src/lib/user-features.ts#L52), do we need to store `AdFreeCookieReasons` in `localStorage` at all?

Reply:
Good point both! I've updated the logic and tests. I'll test on CODE again tomorrow 🙂